packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/shared/chat_wrappers.py

```diff
@@ -371,9 +371,26 @@ def _set_completions(span, choices):
             span, f"{prefix}.finish_reason", choice.get("finish_reason")
         )
 
+        if choice.get("content_filter_results"):
+
+            def _cast_into_span(span, prefix, object):
+                print("caster run")
+                print(prefix)
+                print(object)
+                for key, item in object.items():
+                    if isinstance(item, dict):
+                        _cast_into_span(span, prefix + f".{key}", item)
+                    else:
+                        _set_span_attribute(span, f"{prefix}.{key}", item)
+
+            for category, item in choice.get("content_filter_results").items():
+                # This handels known possible outcomes including filtering errors
+                _cast_into_span(span, f"{prefix}.content_filter_results.{category}", item)
```

Comment:
I think it's better to just send everything in a stringified JSON in the `content_filter_results` attribute

Reply:
now it is and the tests are updated